src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -446,7 +446,9 @@ internal void InsertGraph()
         private string DynamoGraphFromMDFilePath(string path)
         {
             path = HttpUtility.UrlDecode(path);
-            return Path.Combine(Path.GetDirectoryName(path), Path.GetFileNameWithoutExtension(path)) + ".dyn";
+            var rootLevelDir = Path.GetDirectoryName(path);
+            var imagesLocation = Path.Combine(new DirectoryInfo(rootLevelDir).Parent.Parent.FullName, DocumentationBrowserView.SharedDocsDirectoryName);
```

Comment:
I forgot to mention, would you test when you have time if this path is the same when running as DynamoRevit?

Reply:
I just tested and this is the path used by DocumentationBrowser for shared images/dyn files when using Revit and Dynamo 2.19.3
`Revit 2024\AddIns\DynamoForRevit\NodeHelpSharedDocs`
![RevitInsertingGraphDYN](https://github.com/DynamoDS/Dynamo/assets/61755417/2191b0c4-036d-402a-b25b-a05fb7721f35)

Reply:
Thanks, we are good then